src/System.Private.Xml/tests/Xslt/XslTransformApi/CXslTransform.cs

```diff
@@ -2928,6 +2747,13 @@ public void var4(InputType inputType, ReaderType readerType, TransformType trans
         }
     }
 
+    internal sealed class AllowDefaultResolverContext : IDisposable
```

Comment:
I see `CollectionBehavior` already added to XunitAssemblyAttribute.cs, assuming that it is all that is needed.